pkg/ext-proc/backend/metrics.go

```diff
@@ -89,18 +89,21 @@ func promToPodMetrics(metricFamilies map[string]*dto.MetricFamily, existing *Pod
 	}
 	*/
 
+	// TODO: Read from vLLM metrics once the is available.
+	updated.MaxActiveModels = 4
```

Comment:
This is now available in vLLM metrics, @coolkp added it and will work on reading it from vllm

Reply:
Added follow up issue https://github.com/kubernetes-sigs/llm-instance-gateway/issues/22